pyproject.toml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[tool.isort]
+profile = "black"
+line_length = 119
+ignore_whitespace = true
+py_version = 38
+known_first_party = ["nemo"]
+known_third_party = ["nemo_text_processing", "examples", "scripts"]
+sections = ["FUTURE", "STDLIB", "THIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
+default_section = "THIRDPARTY"
+extend_skip = ["setup.py", "docs/source/conf.py"]
+
+
+[tool.pytest.ini_options]
+# durations=0 will display all tests execution time, sorted in ascending order starting from from the slowest one.
+# -vv will also display tests with durration = 0.00s
+addopts = "--verbose --pyargs --durations=0"
+testpaths = ["tests"]
+norecursedirs = [
+    "nemo",
+    "nemo_text_processing",
+    "external",
+    "examples",
+    "docs",
+    "scripts",
+    "tools",
+    "tutorials",
+    "*.egg",
+    ".*",
+    "_darcs",
+    "build",
+    "CVS",
+    "dist",
+    "venv",
+    "{arch}"
+]
+markers = [
```

Comment:
Is this just docstring for the markers ? Where is the logic for markers still placed ? If it's in conf, can you add comment to update this docstring too if we create more markers

Reply:
This is just a port for `markers` from `setup.cfg`, colon is the delimiter for `marker: docstring`
You can see all these markers by executing `pytest --markers`

Reply:
Also, I think it would be good to add permanently `--strict-markers` (to `addopts`) to mark tests with unregistered markers as failed (now it's just a warning that the marker is not registered, e.g there is a typo `@pytest.mark.uint` instead of `unit`). 

Would it be ok?

Reply:
Sounds okay to me

Reply:
Ok, I added `--strict-markers`

Reply:
Yeah makes sense to make it strict.